Update multi-key.md (#426)
BitcoinDesign/Guide: guide/private-key-management/multi-key.md

In the case of a shared wallet, different people will control the keys. The number of keys and required co-signers will depend on the use case. With spouses sharing a *joint account*, a simple 1-of-2 multi-key setup might suffice, meaning there are two keys but only one is required to sign for a transaction to be valid. At the other end of a spectrum a company might require a more complex 3-of-5 setup, requiring three of the five co-signers to approve any transaction.

> Since this is a common misunderstanding, we would like to remember that the usage of the multiple keys is necessary only for signing outgoing transactions, but not for receiving funds.

Reviewer: ```suggestion
It is important to remember that the usage of multiple keys is necessary only for signing outgoing transactions, but not for receiving funds. This is a common misunderstanding.
```

Author: You are absolutely right! Thank you very much @sbddesign .
